packages/compass-global-writes/src/components/states/shard-key-correct.tsx

```diff
@@ -0,0 +1,200 @@
+        <Body>
+          Start querying your database with some of the most{' '}
+          <Link
+            href="https://www.mongodb.com/docs/atlas/global-clusters"
+            hideExternalIcon
+          >
+            common commands
+          </Link>{' '}
+          for Global Writes.
+        </Body>
+        <Body>
+          Replace the text to perform operations on different documents. US-NY
+          is an ISO 3166 location code referring to New York, United States. You
+          can look up other ISO 3166 location codes below.
+        </Body>
+      </div>
+
+      <div className={codeBlockContainerStyles}>
+        <Label htmlFor="finding-documents">Finding documents</Label>
+        <Code language="js" data-testid="sample-finding-documents">
+          {sampleCodes.findingDocuments}
+        </Code>
+      </div>
+
+      <div className={codeBlockContainerStyles}>
+        <Label htmlFor="inserting-documents">Inserting documents</Label>
+        <Code language="js" data-testid="sample-inserting-documents">
+          {sampleCodes.insertingDocuments}
+        </Code>
+      </div>
+
+      <Subtitle>Location Codes</Subtitle>
+      <div className={paragraphStyles}>
+        <Body>
+          Each document’s first field should include an ISO 3166-1 Alpha-2 code
+          for the location it belongs to.
+        </Body>
+        <Body>
+          We also support ISO 3166-2 subdivision codes for countries containing
+          a cloud provider data center (both ISO 3166-1 and ISO 3166-2 codes may
+          be used for these countries). All valid country codes and the zones to
+          which they map are listed in the table below. Additionally, you can
+          view a list of all location codes{' '}
+          <Link href="/static/atlas/country_iso_codes.txt">here</Link>.
+        </Body>
+        <Body>
+          {atlasMetadata?.projectId && atlasMetadata?.clusterName && (
+            <>
+              Locations’ zone mapping can be changed by navigating to this
+              clusters{' '}
+              <Link
+                href={`/v2/${atlasMetadata?.projectId}/clusters/edit/${atlasMetadata?.clusterName}`}
+              >
+                Edit Configuration
+              </Link>{' '}
+              page and clicking the Configure Location Mappings’ link above the
+              map.
+            </>
+          )}
+        </Body>
+      </div>
+
+      <ShardZonesTable shardZones={shardZones} />
+
+      <Subtitle>Unmanage this collection</Subtitle>
+      <Body>
+        Documents belonging to this collection will no longer be distributed
+        across the shards of your global clusters.
+      </Body>
+      <div>
+        <Button
+          data-testid="shard-collection-button"
+          onClick={onUnmanageNamespace}
+          disabled={isUnmanagingNamespace}
+          variant="primary"
+          leftGlyph={
+            isUnmanagingNamespace ? (
+              <SpinLoader title="Unmanaging collection" />
```

Comment:
Pretty sure leafygreen buttons have the loading state support now, maybe we can use that instead so that we're mor aligned with design system?

Reply:
thanks!